addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/HashDisclosureScanRule.java

```diff
@@ -272,7 +272,9 @@ private AlertBuilder buildAlert(String evidence, HashAlert hashAlert) {
                 .setSolution(Constant.messages.getString(MESSAGE_PREFIX + "soln"))
                 .setReference(Constant.messages.getString(MESSAGE_PREFIX + "refs"))
                 .setEvidence(evidence)
-                .setCweId(200) // Information Exposure,
+                .setCweId(
+                        497) // CWE-497: Exposure of Sensitive System Information to an Unauthorized
+                // Control Sphere
```

Comment:
Really better add the comments above, which avoid these break lines all over the place.